wordpress-mobile/WordPress-iOS
# [Authenticator] Fix iOS 13 modal presentation style

Refs #12211. This PR pulls in the latest authenticator pod to fix the modal presentation style for authenticator views. In iOS 13 the default modal presentation style changes to `UIModalPresentationAutomatic`, which resolves as `pageSheet`. This results in the login views being dismissible. I've updated this to `fullScreen`, restoring the previous behaviour.

![before-after-nux](https://user-images.githubusercontent.com/4780/63217321-3604e600-c13c-11e9-9d62-abcb765009d2.png)

**To test:**

* Build and run on iOS 13
* Do a fresh install and verify that the NUX is presented fullscreen and isn't dismissible by swiping down
* Log into a self-hosted site
* In the sites list, tap + and choose 'add self-hosted site'. Verify that the presented view is fullscreen.
* Head to the Me tab and choose Log In. Verify that the login view is presented fullscreen.

Update release notes:
